contrib/completion/MAINTAINERS

```diff
@@ -1,2 +1,3 @@
 Tianon Gravi <admwiggin@gmail.com> (@tianon)
 Jessie Frazelle <jess@docker.com> (@jfrazelle)
+Vincent Bernat <vincent@bernat.im> (@vincentbernat)
```

Comment:
No problem, I'll tell them to just put exec zsh on the top of their configuration file. ;-)

I don't really know what it means to be here. Just telling I will not bothered to be pinged if there is some change or problem to the ZSH completion.